jaypeasee/sorting-hat-quiz
# Feature - UI Details

#### What’s this PR do?
This PR cleans up some specific UI details that were necessary for turning in the project. It includes:
1. Styling the page in multiple breakpoints so that it looks nice in tablet and mobile as well as desktop.
2. Updates the page title.
3. Conditionally renders Error component when state is not updated in Question.
4. Fixes background color height issue on Result.

#### Where should the reviewer start?
* Review media queries in stylesheets for all Components except `App`.
* Review stylesheet in `Result` to see how `.gryffindor`, `.hufflepuff`, `.slytherin`, and `.ravenclaw` are now styled with a mixin
* Review `Question.js` to see the conditional rendering of `Error`.


#### How should this be manually tested?
1. Run `npm start`
2. Confirm the page title has changed in the top tab.
3. Do a squishy test in all views to see the responsiveness and the height color in `Result` fixed.

#### What are the relevant tickets?
* #31 
* #30 
* #29 
* #27 
* #4
